jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -49,16 +49,15 @@
     @Nullable private ImageReference targetImageReference;
     @Nullable private String targetImageCredentialHelperName;
     @Nullable private RegistryCredentials knownTargetRegistryCredentials;
-    @Nullable private String mainClass;
     private ImmutableList<String> javaArguments = ImmutableList.of();
-    private ImmutableList<String> jvmFlags = ImmutableList.of();
     private ImmutableMap<String, String> environmentMap = ImmutableMap.of();
     private ImmutableList<Port> exposedPorts = ImmutableList.of();
     private Class<? extends BuildableManifestTemplate> targetFormat = V22ManifestTemplate.class;
     @Nullable private CacheConfiguration applicationLayersCacheConfiguration;
     @Nullable private CacheConfiguration baseImageLayersCacheConfiguration;
     private boolean allowHttp = false;
-    @Nullable private LayerConfiguration extraFilesLayerConfiguration;
+    private ImmutableList<LayerConfiguration> layerConfigurations = ImmutableList.of();
+    private ImmutableList<String> entrypoint = ImmutableList.of();
```

Comment:
This is very much a nit, so feel free to ignore, but for organization it'd be nice to put `entrypoint` next to `javaArguments` instead of at the end.

Reply:
Sounds good

Reply:
(This same ordering happens in a bunch of other places too, but I didn't want to overload the comments.)

Reply:
Yea, maybe we can leave that to another PR that can set up a nice ordering pattern for all of these parameters.